packages/select/src/Select.tsx

```diff
@@ -73,18 +74,22 @@ export default function Select({
   darkMode = false,
   size = Size.Default,
   disabled = false,
-  usePortal = true,
   placeholder = 'Select',
   className,
   id: idProp,
   label,
+  'aria-labelledby': ariaLabelledBy,
```

Comment:
We should be consistent about this. I don't feel sooo strongly if the B should or shouldn't be capitalized (I've been using lc because it more closely parallels the html attribute) but just a nit that we should try and standardize

Reply:
I didn't change this in this PR, but I think my PR's changes to the props caused your naming change to be reverted. I'll fix.

Reply:
thank you

Reply:
Done